### PR TITLE
Python: Add consistency checks to all data-flow test folders

### PR DESCRIPTION
**Pull request by yoff**
We can do this, since they now all pass :-)

### CODE REVIEW

**Review by tausbn — DISMISSED** [review]
Brilliant!

File: python/ql/test/experimental/dataflow/tainttracking/dataflow-consistency.ql [hunk 1]
```diff
@@ -0,0 +1 @@
+import semmle.python.dataflow.new.internal.DataFlowImplConsistency::Consistency
```
Comment (tausbn) [inline python/ql/test/experimental/dataflow/tainttracking/dataflow-consistency.ql:1]: `experimental/dataflow/tainttracking` isn't a test directory, so this and its companion file are superfluous.

Comment (yoff) [reply python/ql/test/experimental/dataflow/tainttracking/dataflow-consistency.ql]: Thanks, it looks like it made the test fail...

**Review by tausbn — APPROVED** [review]
Nice!

**Pull request merged**